### PR TITLE
Add e2e test for VAMC locations page

### PR DESCRIPTION
**Pull request by mpelzsherman**
Resolves https://github.com/department-of-veterans-affairs/va.gov-team/issues/28283

### CODE REVIEW

File: src/site/layouts/tests/vamc/location_detail.cypress.spec.js [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+    });
+  cy.get('.va-introtext').should('exist');
+  cy.get('a.usa-button').contains('Make an appointment');
+  cy.get('a.usa-button').contains('Register for care');
+  cy.get('a.usa-button').contains('Pharmacy');
+  if (isMobile) {
+    cy.get('#sidenav-menu').should('be.visible');
+  } else {
+    cy.get('#sidenav-menu').should('not.be.visible');
+  }
```
Comment (Rleahy22) [inline src/site/layouts/tests/vamc/location_detail.cypress.spec.js:23]: nit: If this is an expectation on multiple pages, we could add a cypress command for this.

**Review by Rleahy22 — APPROVED** [review]
Just one tiny suggestion, but lgtm 🚀

**Review by gescobardev — APPROVED** [review]

File: src/site/layouts/tests/vamc/location_detail.cypress.spec.js (hunk 1 above)
Comment (mpelzsherman) [reply src/site/layouts/tests/vamc/location_detail.cypress.spec.js:23]: Good idea, thanks!

File: src/site/layouts/tests/vamc/commands.cypress.js [hunk 2]
```diff
@@ -0,0 +1,7 @@
+Cypress.Commands.add('checkSideNav', isMobile => {
```
Comment (Rleahy22) [inline src/site/layouts/tests/vamc/commands.cypress.js:1]: 👍

**Pull request merged**